shanoir-ng-datasets/src/main/java/org/shanoir/ng/configuration/amqp/RabbitMQConfiguration.java

```diff
@@ -30,15 +30,28 @@
  * RabbitMQ configuration.
  */

```

Comment:
This has to be merged in the unique big rabbitMQ configuration made in MS-commons